deploy/charts/arcadia/charts/dataprocess/templates/rbac.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+    name: {{ .Release.Name }}-dataprocess
+    namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ .Release.Name }}-dataprocess
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ .Release.Name }}-dataprocess
+subjects:
+  - kind: ServiceAccount
+    name: {{ .Release.Name }}-dataprocess
+    namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+    name: {{ .Release.Name }}-dataprocess
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+    - versioneddataset
+  verbs:
+    - get
+    - list
+- apiGroups:
+    - ""
+  resources:
+    - datasources
```

Comment:
`datasources `  and `versioneddataset` should use apiGroup `arcadia.kubeagi.k8s.com.cn`